templates/partials/views/settingsview.php

```diff
@@ -178,6 +178,16 @@ class="icon-delete reset-button" id="reset-podcasts" ng-click="resetPodcasts()"/
 			</dl>
 		</div>
 	</div>
+	<h2 translate>Scrobbler Integration</h2>
+	<div>
+		<p ng-hide="settings.scrobbler.configured" translate>The server admin can configure your server for scrobbling. The <strong>Admin</strong> section below details how to achieve this.</p>
+		<p ng-show="settings.scrobbler.configured" translate >The server admin has configured your server for scrobbling.</p>
+		<p ng-show="settings.scrobbler.configured && !settings.scrobbler.hasSession">Click the <strong>Get Scrobble Session</strong> button to send your streams to {{settings.scrobbler.service}}.</p>
+		<p ng-show="settings.scrobbler.configured && settings.scrobbler.hasSession">Click the <strong>Clear Session</strong> to stop scrobbling.</p>
```

Comment:
When there is a session, the text should state clearly something like "Your plays are being scrobbled to Last.fm".